source/structured-streaming.txt

```diff
@@ -358,26 +509,24 @@ Stream to MongoDB from a CSV File
        content: |
 
          To create a :ref:`write stream <write-structured-stream>` to 
-         MongoDB from a ``.csv`` file, first create a `DataStreamReader <https://spark.apache.org/docs/latest/api/scala/org/apache/spark/sql/streaming/DataStreamReader.html>`__ 
-         from the ``.csv`` file, then use that ``DataStreamReader`` to 
+         MongoDB from a CSV file, first create a `DataStreamReader <https://spark.apache.org/docs/latest/api/scala/org/apache/spark/sql/streaming/DataStreamReader.html>`__ 
```

Comment:
Issue:
1) I'm not sure how to get a DataStreamWriter from a DataStreamReader.
2) This sentence contains too many actions. The info could also be easier to follow if corresponded to the flow of data.

Suggestion:
Change sentence to something similar to the following:
To stream data from a CSV file to MongoDB:
1) Create a DataStreamReader that reads from the CSV file
2) Create a DataStreamWriter from ?
3) Call the start() method on the DataStreamWriter instance to begin the stream.